operator/operator.php

```diff
@@ -77,9 +111,13 @@ public function set_user_categories($forum_id)
 	 */
 	public function get_cookie_categories()
 	{
-		// TODO
-		// 1.gets categories from the cookie (will need to be un-encoded by json decode and htmlspecialchars decode)
-		// 2.return categories or an empty array
+		// Get categories from the cookie (htmlspecialchars_decode and json_decode apply)
+		// Return the categories array or an empty array if there were not categories found
+		$cookie_data = htmlspecialchars_decode($this->request->variable($this->config['cookie_name'] . '_ccat', '', true, \phpbb\request\request_interface::COOKIE));
```

Comment:
Just a 1-liner comment please:
// Get categories from the cookie and htmlspecialchars decode it